packages/zosfiles/src/methods/create/Create.ts

```diff
@@ -272,10 +272,18 @@ export class Create {
                         // F, V, or U are required; B, A, M, S, T or additional
                         // VBA works on mainframe but not via zOSMF
                         switch (tempOptions.recfm.toUpperCase()) {
+                            case "D":
```

Comment:
Please add unit tests to cover the updated lines. The unit tests for this file are in `packages/zosfiles/__tests__/__unit__/methods/create/Create.unit.test.ts`

You could add a test that validates all the "recfm" values similar to this one for testing "dsntype":
https://github.com/zowe/zowe-cli/blob/master/packages/zosfiles/__tests__/__unit__/methods/create/Create.unit.test.ts#L101-L119